openshift/qeb-hwt-template.yaml

```diff
@@ -55,22 +55,6 @@ parameters:
     displayName: "Thoth Host"
     required: true
 
-  - name: IMAGE_STREAM_REGISTRY
```

Comment:
do we have some CD to create the ImageStreamTag in the namespace where we need it?

Reply:
Yes we have it in adviser